addons/crm/views/crm_lead_views.xml

```diff
@@ -731,7 +716,108 @@
                             'default_use_template': True,
                             'default_template_id': ref('crm.email_template_opportunity_mail'),
                         }"
-                />
+        />
+
+        <act_window name="Send email"
+                res_model="mail.compose.message"
+                binding_model="crm.lead"
+                binding_views="list"
+                view_mode="form"
+                target="new"
+                id="crm.action_lead_single_mail"
```

Comment:
Rename id to something like crm_lead_act_window_compose in order to follow guidelines ?